contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -207,13 +210,13 @@ In this case, only the `root` component can be accessed in Vue while the rest ar
 
     ```js
     watch: {
-        totalCount: function() {
-            let result = 0
-            this.items.forEach((item) => result += item.count);
-            this.totalCount = result;
+      totalCount: function() {
+        let result = 0
+        this.items.forEach((item) => result += item.count);
```

Comment:
```suggestion
        this.items.forEach((item) => result += item.count);
```

Not Vue related, but can be more idiomatic using `Array.prototype.reduce` to sum instead,